### ISSUE TITLE
Add option to add OSM Notes

### ISSUE THREAD

**Issue by iiitomo** [organicmaps/organicmaps#3918]
Thx for this lovely app.
While navigating (or not), I wish I could long-press anywhere on the map, then press an 'Add a note' button, fill the OSM Note description, and validate.

To be discussed: this feature could be available only after activation in the Options. And/or could be available only when logged to OSM account.

**Comment by Jean-BaptisteC** [comment]
Duplicate of #266
